### PR TITLE
feat: add geo.point support

### PR DESCRIPTION
**Pull request by iloveitaly**
related to #22

### CODE REVIEW

**Comment by fuelen** [comment]
Thank you for the PR.

Could you add `geo` (or `geo_postgis`??) as an optional dependency and a couple of tests?

**Comment by iloveitaly** [comment]
done!

File: mix.exs [hunk 1]
```diff
@@ -41,10 +41,13 @@ defmodule Ecto.DevLogger.MixProject do
...
       {:ecto, "~> 3.7"},
+      {:jason, "~> 1.0"},
+
+      {:ex_doc, "~> 0.28", only: :dev, runtime: false},
+
       {:ecto_sql, "~> 3.7", only: :test},
       {:postgrex, "~> 0.16", only: :test},
-      {:jason, "~> 1.0"},
-      {:ex_doc, "~> 0.28", only: :dev, runtime: false}
+      {:geo, "~> 3.5.1", only: :test},
```
Comment (fuelen) [inline mix.exs:50]: please make it `optional` and not only for `:test`

File: test/ecto/dev_logger/printable_parameter_test.exs [hunk 2]
```diff
@@ -88,6 +88,9 @@ defmodule Ecto.DevLogger.PrintableParameterTest do
              %Postgrex.INET{address: {127, 0, 0, 1}, netmask: nil}
            ]) == "'{127.0.0.1/24,127.0.0.1}'"
 
+    assert to_expression([%Geo.Point{coordinates: {44.21587, -87.5947}, srid: 4326, properties: %{}}]) ==
+      "'{\"%Geo.Point{coordinates: {44.21587, -87.5947}, properties: %{}, srid: 4326}\"}'"
```
Comment (fuelen) [inline test/ecto/dev_logger/printable_parameter_test.exs:92]: This is not a valid string representation of geo point.
I didn't use postgis, so I can't help here, but quick googling shows, that it should be represented like `SRID=4326;POINT(44.21587 -87.5947)`

**Review by fuelen — CHANGES_REQUESTED** [review]

**Comment by javiercr** [comment]
Heads up! I've open new PR to finish the work started here!

**Pull request closed without merging**